bootstrap/api/swagger.yaml

```diff
@@ -0,0 +1,133 @@
+swagger: "2.0"
+info:
+  description: "Deployment API."
+  version: "1.0.0"
+  title: "Kubeflow Deployment"
+  termsOfService: "https://www.kubeflow.org/"
+  contact:
+    email: "kubeflow-engineering@google.com"
+  license:
+    name: "Apache 2.0"
+    url: "http://www.apache.org/licenses/LICENSE-2.0.html"
+host: "https://www.kubeflow.org/"
+basePath: "/kfctl"
+tags:
+- name: "deployment"
+  description: "A kubeflow deployment on GCP"
+  externalDocs:
+    description: "Find out more"
+    url: "https://www.kubeflow.org/docs/started/getting-started-gke/"
+schemes:
+- "https"
+paths:
+  /e2eDeploy:
```

Comment:
Should we have a crud style API?  

We could model it on a combination of a K8s & GCP
https://cloud.google.com/deployment-manager/docs/reference/latest/deployments/insert

So should we have a create request at a URL like
/projects/${PROJECT}/deployments

And then a get at
/projects/${PROJECT}/deployments/${DEPLOYMENTNAME}